modules/statistics/AvgRTPStatsReporter.js

```diff
@@ -516,6 +516,40 @@ export default class AvgRTPStatsReporter {
             = new AverageStatReport('stat_avg_framerate_screen_local');
 
         /**
+         * Average pixels for remote screen streaming videos (reported only if
+         * not a <tt>NaN</tt>).
+         * @type {AverageStatReport}
+         * @private
+         */
+        this._avgRemotePixels
+            = new AverageStatReport('stat_avg_pixels_remote');
+
+        /**
+         * Average pixels for remote screen streaming videos (reported only if
+         * not a <tt>NaN</tt>).
+         * @type {AverageStatReport}
+         * @private
+         */
+        this._avgRemoteScreenPixels
+            = new AverageStatReport('stat_avg_pixels_screen_remote');
+
+        /**
+         * Average pixels for local video (camera)
+         * @type {AverageStatReport}
+         * @private
+         */
+        this._avgLocalPixels = new AverageStatReport('stat_avg_pixels_local');
```

Comment:
`_avgLocalCameraPixels`? it'd make it more obvious when reading it in code (instead of having to go back to the comment)